e2e/artefacts/k8s/vsp-ds.yaml

```diff
@@ -11,6 +11,15 @@ spec:
       labels:
         name: vsp
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node-role.kubernetes.io/control-plane
+                operator: DoesNotExist
+              - key: node-role.kubernetes.io/master
+                operator: DoesNotExist
```

Comment:
Does this mean that the VSP will get on nodes that do not have control-plane and master roles?

Reply:
Yes, we want to avoid it to be deployed on master nodes. Later we can use some node labels(using NFD) to select only nodes with IPUs.